ibm/mas_devops/plugins/filter/test_filters.py

```diff
@@ -0,0 +1,70 @@
+# -----------------------------------------------------------
```

Comment:
Errrrrrm, I hope this is just a mistake of putting the wrong license file in place, rather than copying product code into this repo? 

ansible-devops is open source code, licensed under the EPL, we can not published licensed materials in this repository